Integrations/TrendMicroApex/TrendMicroApex.py

```diff
@@ -0,0 +1,239 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+import urllib3
+import jwt
+import base64
+import hashlib
+import time
+import json
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+''' CLIENT CLASS'''
+
+USDOAPIPATH = '/WebApp/api/SuspiciousObjects/UserDefinedSO'
+PRODAGENTAPIPATH = '/WebApp/API/AgentResource/ProductAgents'
+
+
+class Client(BaseClient):
+    def __init__(self, url_base, api_key, app_id, verify, proxy):
+        self.url_base = url_base
+        self.api_key = api_key
+        self.application_id = app_id
+
+        BaseClient.__init__(self, url_base, verify, proxy, ok_codes=None, headers=None, auth=None)
```

Comment:
any reason to pass here `None` to the last 3 args?
any reason not to use `super().__init__` ?